packages/neptune-extension/src/common/components/checkout-modal/CheckoutModal.less

```diff
@@ -0,0 +1,5 @@
+.checkout-modal {
+  &__input {
```

Comment:
Would you mind refactoring this to one `Input` component ? We already have such quickfix code in every modal that uses input.

Reply:
I dont think Input by default should have any flex properties.

Reply:
This depends, if the `flex` props is needed to display the input properly, than definitely it should be moved there. But if it's needed only in some case, then this case should be factored out.

What is the reason to use `flex` here ?

Reply:
Flex is needed to be displayed properly only in selected cases (flex-form).
I can imagine using Input outside of flex container and there flex is not needed